test/multistage-registry/registry/ipi/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
Could we not re-use these folders? The fact that they are used within multiple tests makes any changes here very hard and IMHO everything needed for any test should only be used by that test (Copying everything to a new dir is totally fine)